src/skins-qt/menus.cc

```diff
@@ -180,6 +180,8 @@ static const audqt::MenuItem playlist_remove_items[] = {
 };
 
 static const audqt::MenuItem playlist_select_items[] = {
+    audqt::MenuCommand ({N_("Search and Select"), "edit-find", "CTRL+F"}, action_playlist_search_and_select),
```

Comment:
The other entries have "Ctrl" in camel-case (not "CTRL"). Not sure if it matters.

Reply:
It doesn't matter, but I prefer consistency as well. Fixed.